doc/start/hardware-recommendations.rst

```diff
@@ -318,7 +324,7 @@ media cost.  Moreover, when using NVMe SSDs, you do not need *any* HBA.  This
 additionally reduces the HDD vs SSD cost gap when the system as a whole is
 considered. The initial cost of a fancy RAID HBA plus onboard cache plus
 battery backup (BBU or supercapacitor) can easily exceed more than 1000 US
```

Comment:
By the way, modern RAID controllers have moved to non-volatile cache technologies that don't require battery backup. But I have no opinion on whether this should be updated.